modules/core/src/main/java/org/apache/ignite/internal/processors/odbc/odbc/OdbcConnectionContext.java

```diff
@@ -117,7 +117,8 @@ public OdbcConnectionContext(GridKernalContext ctx, GridNioSession ses, GridSpin
     }
 
     /** {@inheritDoc} */
-    @Override public void initializeFromHandshake(ClientListenerProtocolVersion ver, BinaryReaderExImpl reader)
+    @Override public void initializeFromHandshake(GridNioSession ses,
```

Comment:
If we pass session instance here then we don't need it a as class field anymore. Same for the JDBC.

Reply:
Will do.